pyiron_continuum/damask/damaskjob.py

```diff
@@ -110,6 +110,12 @@ def _load_results(self, file_name="damask_loading.hdf5"):
         self.output.strain = self.average_spatio_temporal_tensors('epsilon_V^0.0(F)')
         self.output.stress_von_Mises = self.average_spatio_temporal_tensors('sigma_vM')
         self.output.strain_von_Mises = self.average_spatio_temporal_tensors('epsilon_V^0.0(F)_vM')
+    
+    def writeresults2vtk(self):
+        """
+            save results to vtk files
+        """
+        self.output.damask.export_VTK()
```

Comment:
It is desirable to export the VTK files into `self.working_directory`. There might be a workaround. You can change the current working directory to `self.working_directory` and the after exporting the vk files, return to the old working directory.